index.html

```diff
@@ -122,6 +122,40 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+        <!-- ________ Sappenfield's Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Albert J. Sappendield</p>
+          <p class="contact">
+            <i class="fab fa-x-twitter"></i>
+            <a href="https://twitter.com/aj_sappenfield" target="_blank">aj_sappenfield</a>
+            <i class="fab fa-linkedin"></i>
+            <a href="http://www.linkedin.com/in/a-j-sappenfield/" target="_blank">AJ Sappenfield</a>
+            <i class="fab fa-github"></i>
+            <a href="https://www.github.com" target="_blank">sappenfield</a>
+          </p>
+          <p class="about">Enthusiastic and driven computer science student with a diverse background 
+            in technical problem-solving, project management, and cross-functional collaboration.
+            Knowledgeable in Java, C++, and SQL programming languages, coupled with a track record of developing
+            innovative solutions and leading successful multi-channel marketing campaigns. Seeking to leverage technical 
+            expertise and project management skills to contribute to groundbreaking advancements in aerospace technologies. 
+            </p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://stackoverflow.com/" target="_blank" title="First Resource">Stack Overflow ftw</a>
+              </li>
+              <li>
+                <a href="https://leetcode.com/problems/two-sum/" target="_blank" title="Second Resource">Leetcode a day keeps the depression away</a>
```

Comment:
Not sure if this is intentional or not (it should be fine either way) but the link here goes directly to the two sum problem, as opposed to the landing page for leetcode. Feel free to fix that if it's a mistake, or leave it if it isn't.

Reply:
It was intentional, I find that specific problem is an important intro into algorithms lol

Reply:
Awesome, figured I'd check.